File: backend2/src/main/java/com/luizgasparetto/ecommerceag2/dto/PurchaseMessage.java
package com.luizgasparetto.ecommerceag2.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class PurchaseMessage {
     private String firstName;
     private String lastName;
     private BigDecimal valor;
     private List<CartItemDto> cartItems;
     private String cpf;
     private String country;
     private String cep;
     private String address;
     private String number;
     private String complement;
     private String district;
     private String city;
     private String state;
     private String phone;
     private String email;
     private String note;
     private String delivery;
     private String payment;
}

File: backend1/src/main/kotlin/com/luizgasparetto/backend1/dto/PurchaseMessage.kt
package com.luizgasparetto.backend1.dto

import java.math.BigDecimal

data class PurchaseMessage(
    val firstName: String,
    val lastName: String,
    val cpf: String,
    val country: String,
    val cep: String,
    val address: String,
    val number: String,
    val complement: String,
    val district: String,
    val city: String,
    val state: String,
    val phone: String,
    val email: String,
    val note: String,
    val delivery: String,
    val payment: String,
    val valor: BigDecimal,
    val cartItems: List<CartItemDto>
)
